build.gradle

```diff
@@ -207,15 +206,17 @@ allprojects {
 
         // Call toString explicitly to avoid lazy evaluation
         jdkImageName = "${os}-${jdkArch}-normal-server-${correttoDebugLevel}".toString()
+        archSuffix = is_musl ? "-musl" : ""
+        headless = is_headless ? "-headless" : ""
         // no debug level suffix for release build
         if (correttoDebugLevel == "release") {
-            correttoJdkArchiveName = "amazon-corretto-${project.version.full}-${os}-${correttoArch}".toString()
+            correttoJdkArchiveName = "amazon-corretto${headless}-${project.version.full}-${os}-${correttoArch}${archSuffix}".toString()
```

Comment:
I am a little concerned about changing the naming convention like this. Downstream consumers and likely some of our release infrastructure expects "Corretto-MV" so we may want to add this at the end of the file name.

Reply:
Agree, I'll put it in the end after archSuffix. 
For non release debugLevel - after archSuffix in front of correttoDebugLevel, just in case someone expects the name to end with debugLevel.